code/controllers/subsystem/persistence.dm

```diff
@@ -253,3 +267,11 @@ SUBSYSTEM_DEF(persistence)
 	file_data["data"] = saved_modes
 	fdel(json_file)
 	WRITE_FILE(json_file, json_encode(file_data))
+
+/datum/controller/subsystem/persistence/proc/CollectAntagReputation()
+	var/json_file = file("data/AntagReputation.json")
+	var/list/file_data = list()
+	file_data["data"] = antag_rep
+	fdel(json_file)
+	WRITE_FILE(json_file, json_encode(file_data))
```

Comment:
instead of these 2 and the file call above. Just use text2file

Reply:
Should the other calls like chisel's `var/json_file = file("data/npc_saves/ChiselMessages[SSmapping.config.map_name].json")` in this file also be changed?

Reply:
only if you want